kb/deadline-exceeded-troubleshooting.md

```diff
@@ -0,0 +1,115 @@
+---
+slug: deadline-exceeded-troubleshooting
+title: DeadlineExceeded error troubleshooting
+tags:
+  - kb-article
+date: 2022-11-09T00:00:00Z
+---
+
+All client-side requests made to the [Temporal Cluster](/concepts/what-is-a-temporal-cluster) are [gRPC requests](https://grpc.io/docs/what-is-grpc/core-concepts/#deadlines).
+Sometimes, when these frontend requests can't be completed, you'll see this particular error message: `Context: deadline exceeded`.
+Network interruptions, timeouts, server overload, and Query errors are some of the causes of this error.
+
+The following sections discuss the nature of this error and how to troubleshoot it.
+
+#### Check Frontend Service logs
+
+:::note
+
+Cloud users cannot access some of the logs needed to diagnose the source of the error.
+
+If you're using Temporal Cloud, check your Workflow History for error messages and create a [support ticket](https://support.temporal.io/) with Temporal.
+Provide the full error message in your ticket.
+
+:::
+
+[Frontend Service](/clusters/#frontend-service) logs can show which parts of the Cluster aren't working.
+In order for the error to appear, there has to be a service pod or container up and running.
+
+OSS users can verify that the [Frontend Service](/clusters#frontend-service) is connected and running by using `tctl`.
```

Comment:
```suggestion
For the error to appear, a service pod or container must be up and running.

OSS users can verify that the Frontend Service is connected and running by using `tctl`.
```
First sentence edited per https://developers.google.com/style/word-list#in-order-to.

Link in second sentence removed because it appears in the preceding sentence. (And, yes, I missed that entirely earlier. Mea culpa.)